typings/index.d.ts

```diff
@@ -1106,6 +1108,7 @@ declare module 'discord.js' {
 		public setTimestamp(timestamp?: Date | number): this;
 		public setTitle(title: StringResolvable): this;
 		public setURL(url: string): this;
+		public static checkField(name:string, value:string, inline?:boolean): EmbedField;
```

Comment:
```suggestion
		public static checkField(name: StringResolvable, value: StringResolvable, inline?: boolean): EmbedField;
```
This should use StringResolvable as the function uses util.resolveString